services/aws-s3-storage/src/main/java/com/gentics/mesh/storage/s3/S3BinaryStorageImpl.java

```diff
@@ -142,7 +142,6 @@ public void createBucket(String bucketName) {
 		}
 
 		S3RestResponse s3RestResponse = new S3RestResponse(presignedRequest.url().toString(), presignedRequest.httpRequest().method().toString(), presignedRequest.signedHeaders());
-		s3RestResponse.setVersion(nodeVersion);
```

Comment:
I'm not yet sure if we may omit setting the node version explicitly. Will try keeping it in mind until the tests.